cc/toolchains/gcc_arm_embedded/init.bzl

```diff
@@ -0,0 +1,10 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+def gcc_arm_embedded_toolchain():
```

Comment:
We should add this function to `cc/respositories.bzl` instead of creating this file.